modules/ROOT/pages/create-an-api-configure-service-mesh-CRD.adoc

```diff
@@ -0,0 +1,84 @@
+= Create an API Using the CRD
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, downloading, installing
+
+After you provision the adapter, you must create an API to bind to the service. The following diagram explains the flow for creating an API:
+
+//image::configure-service-mesh-flowchart-2.png[]
+image::create-API-CRD-flowchart.png[]
+
+You can create an API using Anypoint Platform or Anypoint Service Mesh:
+
+* <<create-api-AP, Create an API using Anypoint Platform>>
++
+If you want to create REST or SOAP APIs with OAS or WSDL definition, you use Anypoint Platform. You can manage your contracts and complex API configurations easily in Anypoint Platform. 
++
+Note that if you want to upload a REST API or OAS asset type, you upload a .json or .yaml file. Alternatively, if you want to upload a SOAP API or WSDL asset type, you upload a .wsdl or .xml file.
+* <<create_api-CRD, Create an API using Anypoint Service Mesh Autocreate>>
++
+The Autocreate feature of Anypoint Service Mesh allows you to create an HTTP API with no contracts or definitions. When you choose this option, Anypoint Service Mesh creates the API for you in Anypoint Platform. You do not have to manually log in to Anypoint Platform.
+
+[[create-api-AP]]
+== Creating an API Using Anypoint Platform
+
+To create an API using Anypoint Platform, perform the following steps:
+
+. Go to *Anypoint Platform > API Manager* and select the *Manage API* drop-down list. 
+. Select *Create new API*.
++
+The Creating an asset page is displayed.
+. Specify the *Name*, and *Asset types* details for the API.  
++
+Your Asset is now created in Exchange for reuse. After you create the Asset, you are now ready to perform Endpoint configuration.
++
+The Endpoint Configuration window is displayed.
+. In the *Managing type* field, select the *Basic Endpoint* radio button.
+. In the *Application type* field, select the *Non-Mule application* radio button.
+. Optionally, specify the *Implementation URI*, and click *Save*.
++
+Your API is now created.
+
+[[create_api-CRD]]
+== Creating an API Using a CRD
+
+To create an API in Anypoint Platform using a CRD:
+
+. Create your adapter definition file and save it with a '.yaml' file extension:
++
+[source,text,linenums]
+----
+apiVersion: servicecatalog.k8s.io/v1beta1
```

Comment:
@tbruno please verify this. it's missing a label

Reply:
Done! Thanks!